test/conformance/kuttl/mutate/clusterpolicy/standard/existing/namespaceselector/02-pod.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- file: pod.yaml
+assert:
+- pod.yaml
```

Comment:
Can we have consistent newlines at the end of files? Seems weird that half have newlines and the other half don't.

Reply:
Sure added!